schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+	id INTEGER PRIMARY KEY NOT NULL,
+	name VARCHAR(50),
+	date_of_birth DATE,
+	escape_attempts INTEGER,
+	neutered BOOLEAN,
+	weight_kg FLOAT(2)
```

Comment:
- [ ] [Optional] It would be better in this case if you used `DECEMAL` instead of `FLOAT`. See this link about [numerical types](https://www.postgresql.org/docs/current/datatype-numeric.html#DATATYPE-FLOAT:~:text=%C2%A0Next-,8.1.%C2%A0Numeric%20Types,-8.1.1.%20Integer%20Types) in Postgres.

Reply:
@Omar-Muhamad 
Thanks for the link, _insightful!_